test/e2e/driver/logging-e2e-specs.js

```diff
@@ -25,7 +24,7 @@ describe('commands - logging', function () {
   describe('getLogTypes', function () {
     it('should get the list of available logs', async function () {
       driver.getLogTypes.should.an.instanceof(Function);
-      (await driver.getLogTypes()).should.eql(_.keys(SUPPORTED_LOG_TYPES));
+      (await driver.getLogTypes()).should.eql(_.keys(driver.supportedLogTypes));
```

Comment:
is keys order always preserved?

Reply:
I think it is, though it's not guaranteed. But this test is not new and hasn't failed before.

But I could change it to how I did it in the android driver (i.e., an exclusive or of the two ought to be empty).